app/views/layouts/application.html.erb

```diff
@@ -5,6 +5,10 @@
     <%= csrf_meta_tags %>
 
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+
+    <!-- BUG: do we really need those extra libs? -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.11/lodash.js"><script>
```

Comment:
nice! we could add google fonts and icons as well

Reply:
done